synapseclient/core/utils.py

```diff
@@ -38,12 +38,14 @@
 BUFFER_SIZE = 8*KB
 
 
-def md5_for_file(filename, block_size=2*MB):
+def md5_for_file(filename, callback=None, block_size=2 * MB):
```

Comment:
Make the new callback the last optional argument in case anyone is already using this function passing in block_size as the 2nd positional argument. If they were adding the new argument as a 3rd arg won't break their usage.